docs/SceneWalkthrough/LobbiesWalkthrough.md

```diff
@@ -0,0 +1,33 @@
+## **Lobbies Demo**
+This demo showcases the lobby and includes a voice chat interface in a combined demo. The demo functions as a waiting room style lobby where users can create a new lobby or join existing ones, once in a lobby users can communicate through the lobby voice chat.
+- The topmost window is where users can create and manage their own lobby
+    - The ``Bucket ID`` Is the name or lobby ID used to find the lobby through search functions
+    - The ``Max Players`` Drowpdown selects the maximum number of user the lobby will be able to hold
+    - The ``Level`` dropdown selects the level the lobby will work on, this is also used for searching for lobbies through search functions
+    - The ``Permission`` dropdown selects how user are able to join the lobby
+        - ``Public`` Allows anyone to join
+        - ``Join via Presence`` Allows only friends to join (Presence is the term used for the entire friends/status interface)
+        - ``Invite Only`` Allows only users who have been sent an invite from a player in the lobby to join
+    - ``Allow Invites`` determines whether or not players in the lobby can invite other players
+    - ``Presence Enabled`` determines whether or not to associate the lobby with the [presence](https://dev.epicgames.com/docs/epic-account-services/eos-presence-interface) system
+    -  ``RTC Voice Room`` determines whether or not the lobby will have voice chat
+    - ``Anti-Cheat Enabled`` toggles Easy-Anti Cheat on and off
```

Comment:
More precisely, enabling the EAC toggle will kick cheating players or leave if the owner is cheating